mlflow_cratedb/adapter/ddl/cratedb.sql

```diff
@@ -0,0 +1,137 @@
+CREATE TABLE IF NOT EXISTS {schema_prefix}"datasets" (
```

Comment:
Good idea with the `search_path`, I think the code in this repository is already using that technique partly through SQLAlchemy.

Concluding that, I may also think, that when using that here again, we would not need to populate the schema name (here: `{schema_prefix}`) into the SQL DDL statements at all, so the code could be simplified?

Reply:
> Concluding that, I may also think, that when using that here again, we would not need to populate the schema name (here: {schema_prefix}) into the SQL DDL statements at all, so the code could be simplified?

Yes right.

Reply:
Improved with 3488790f01446.